tests/testdata/make_test_files.bat

```diff
@@ -0,0 +1,47 @@
+@echo off

```

Comment:
Very crude script that builds msvc samples in-place (instead of in a temp folder like the .sh script)